### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by hadleyrich** [simulot/immich-go#80] [General]
Just a quick note to say thanks for your hard work on this, I did a Google Photos import about 6 months ago using other tools and was never quite happy with the process.

I've just set up a new instance of immich and use your tool instead after discovering it at the bottom of the discussion on the immich repo.

It works fantastically and I'm much more satisfied with the quality of import, I finally feel like I can stop running Google Photos in tandem now :)

Feel free to close this out I just wanted you to know that your work is very much appreciated

**Comment by simulot** [comment]
What a nice issue!

I'm amazed the see how that has started like a joke as got some success!
I'll transform the issue into a discussion, and make it visible.

Thank you very much!

**Comment by huytman** [comment]
Adding another thanks here. Played with Imich in the past - but decided to give it a real chance with a Google Take Out import. Your tool worked great. No issues and I can hardly imagine how the process could be made better. You don't even need to unzip the take out files! Btw, I had 1+tb of files (25x50gb) with about 60k photos. Worked like a charm.

**Comment by rementis** [comment]
This is fantastic software, well done!

**Reply by simulot** [reply]
Thank you.
Glad you find it useful.

**Comment by ajitesh-kuppa-sivakumar** [comment]
Whoa! This tool saved my time. Thanks @simulot for creating this tool and making it open-source. 
Cheers 🥂.

**Comment by rbuchberger** [comment]
Holy hell, thank you so much for building this. This tool took a task I've been dreading for like a year now and made it effortless. It chewed through 20 years of photos like nothing. Well done to the devs, I love you

**Reply by simulot** [reply]
Thank you for your donation! 😄

**Comment by linuxturtle** [comment]
I'll add my thanks!  I've been playing with immich for over a year, dreading the idea of importing my huge archive, and putting it off.  I finally tried doing it on a small archive (26G), and it went so smoothly, with such great results, that I'm just blown away!  Thank you so much!

**Comment by madeofpendletonwool** [comment]
Adding my thanks as well! Moving from version to version is SOOO much faster if things break for whatever reason. This program is seriously awesome.

**Reply by simulot** [reply]
You're welcome 
Thank you

**Comment by Lomax1920** [comment]
Thank you so much for putting this together.  I am way overdo for moving off paid cloud storage.  I had 200gb in Google that this ripped through.

**Comment by gusvendegna** [comment]
Absolutely Beautiful software. Worked perfectly on my first try to import 20GB in no time! TYSM!
